@hts-forking/scripts/json-rpc-mock.js

```diff
@@ -121,6 +121,15 @@ const mirrorNodeClient = {
             noAllowance
         );
     },
+    async getTokenRelationship(idOrAliasOrEvmAddress, tokenId) {
+        this.append('getTokenRelationship', idOrAliasOrEvmAddress, tokenId);
+        const noTokens = { tokens: [] };
```

Comment:
nit: pull `noTokens` out so that it's static and not created on each call.

Reply:
@Nana-EC I'd usually prefer to have definitions in their more specific scope (unless there is another reason to not to). This is to make it clear to see where is being used. If it's in outer scope, I might think it's used in several places, which is not.